aemrules.properties

```diff
@@ -1,11 +1,17 @@
 category=External Analysers
 description=Adds rules for AEM Java development
 homepageUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube
-archivedVersions=0.8,0.9,0.10
-publicVersions=0.11
+archivedVersions=0.8,0.9,0.10,0.11
+publicVersions=1.0
 
 defaults.mavenGroupId=com.cognifide.aemrules
-defaults.mavenArtifactId=aemrules
+defaults.mavenArtifactId=sonar-aemrules-plugin
+
+1.0.description=HTL Support
+1.0.sqVersions=[6.7,LATEST]
+1.0.date=2020-02-07
+1.0.changelogUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/tag/v1.0
+1.0.downloadUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/download/v1.0/sonar-aemrules-plugin-1.0.jar
 
 0.11.description=Fix use of removed dependency, add two rules
 0.11.sqVersions=[6.7,LATEST]
```

Comment:
Only one version can be compatible with `LATEST` at a time. Best practice is to set the range end of the old version to the current SQ version.

Reply:
Fixed.